auth/index.html

```diff
@@ -496,21 +503,22 @@ <h2>Message log</h2>
       /**
        * Converts a `BigInt` into a base64url encoded string
        * @param {BigInt} num
+       * @param {number} length: optional expected length of the resulting buffer
```

Comment:
this doesn't return a buffer, it returns a string. So...maybe this is "number of bytes contained in the resulting string"? Tricky tricky.